### PR TITLE
sonarAnalysis.stages embedded into Jenkinsfile

### PR DESCRIPTION
**Pull request by Ginxo**
**Thank you for submitting this pull request**

<details>
<summary>
How to retest this PR or trigger a specific build:
</summary>

* <b>a pull request</b> please add comment: <b>Jenkins retest this</b>
 
* <b>a full downstream build</b> please add comment: <b>Jenkins run fdb</b>
  
* <b>a compile downstream build</b> please  add comment: <b>Jenkins run cdb</b>

* <b>a full production downstream build</b> please add comment: <b>Jenkins execute product fdb</b>

* <b>an upstream build</b> please add comment: <b>Jenkins run upstream</b>
</details>

### CODE REVIEW

**Comment by Ginxo** [comment]
testing it, please don't merge yet

**Comment by Ginxo** [comment]
tested here https://rhba-jenkins.rhev-ci-vms.eng.rdu2.redhat.com/job/KIE/job/master/job/pullrequest/job/optaweb-vehicle-routing-master.pr/30

File: Jenkinsfile [hunk 1]
```diff
@@ -69,25 +69,15 @@ pipeline {
...
+                    def gitURL = env.ghprbAuthorRepoGitUrl ?: env.GIT_URL
+                    def project = util.getProjectGroupName(util.getProject(gitURL))[1]
+                    if(["optaplanner", "drools", "appformer", "jbpm", "drools-wb", "kie-soup", "droolsjbpm-integration", "kie-wb-common", "openshift-drools-hacep", "optaweb-employee-rostering", "optaweb-vehicle-routing"].contains(project))
+                    {
+                        dir("${env.WORKSPACE}") {
+                            maven.runMavenWithSettingsSonar("771ff52a-a8b4-40e6-9b22-d54c7314aa1e", "-nsu generate-resources -Psonarcloud-analysis", "SONARCLOUD_TOKEN", "sonar_analysis.maven.log")
                         }
-                      }
+                    } else {
+                        println "Project ${project} shouldn't be analyzed"
```
Comment (mareknovotny) [inline Jenkinsfile:80]: I would add more meaningful log message

```suggestion
                        println "Project ${project} shouldn't be analyzed by sonarcloud"
```

**Review by mareknovotny — COMMENTED** [review]
just minor suggestion to the printed log msg

**Review by mbiarnes — APPROVED** [review]
+1

**Comment by Ginxo** [comment]
latest changes tested by https://rhba-jenkins.rhev-ci-vms.eng.rdu2.redhat.com/job/KIE/job/master/job/pullrequest/job/optaweb-vehicle-routing-master.pr/33

**Pull request merged**